app/code/Magento/User/Controller/Adminhtml/Auth/Forgotpassword.php

```diff
@@ -7,26 +7,46 @@
 namespace Magento\User\Controller\Adminhtml\Auth;
 
 use Magento\Security\Model\SecurityManager;
+use Magento\Framework\App\ObjectManager;
+use Magento\Backend\App\Action\Context;
+use Magento\User\Model\UserFactory;
+use Magento\Security\Model\SecurityManager;
```

Comment:
This specific one is now imported twice. Please fix.

Reply:
Fixed.